onnxruntime/core/framework/tensorprotoutils.cc

```diff
@@ -1259,6 +1267,26 @@ ONNX_NAMESPACE::TensorProto TensorToTensorProto(const Tensor& tensor, const std:
     for (; f < end; ++f) {
       *mutable_string_data->Add() = *f;
     }
+  } else if (use_tensor_buffer && tensor.SizeInBytes() > 127) {
```

Comment:
Why 127? It is a magic number. Please add a comment to describe why you chose this number.